### PR TITLE
Adding Hostinger logo to M3 site

### PR DESCRIPTION
**Pull request by gibbscullen**
**What this PR does / why we need it**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing and/or backwards incompatible change?**:

```release-note

```

**Does this PR require updating code package or user-facing documentation?**:

```documentation-note

```

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,6 @@
 [submodule ".ci"]
 	path = .ci
 	url = https://github.com/m3db/ci-scripts.git
+[submodule "site/site/docs-theme"]
```
Comment (ChrisChinchilla) [inline .gitmodules:4]: @gibbscullen None of this theme stuff should be here… Your local git might need some changes, not sure. 🤔

**Review by ChrisChinchilla — CHANGES_REQUESTED** [review]

File: site/static/index.html [hunk 2]
```diff
@@ -802,6 +802,9 @@ <h3 class="font-size-26 text-center font-weight-semibold">
                     <img src="../images/home/logos/trivago_logo.jpg" class="testimonial-logos__logo--trivago lazyload" loading="lazy" alt="trivago_logo.jpg" />
                 </div>
             
+                <div class="testimonial-logos__logo text-center col-md-3 d-flex flex-column justify-content-center align-items-center c-mt-8">
```
Comment (ChrisChinchilla) [inline site/static/index.html:805]: But this is fine :)

File: .gitmodules (hunk 1 above)
Comment (gibbscullen) [reply .gitmodules:4]: @ChrisChinchilla do I need to delete the existing theme in lieu of the new theme?

**Comment by gibbscullen** [comment]
Closing for new PR - making change in GitHub.

**Pull request closed without merging**